src/test/java/nextstep/subway/unit/SectionsTest.java

```diff
@@ -62,6 +63,28 @@ private static Stream<Arguments> duplicateStationParameters() {
         );
     }
 
+    @DisplayName("처음, 가운데, 마지막 구간을 제거한다.")
```

Comment:
테스트 주석은 보통 내가 무슨 기능을 테스트하는지 , 어떤 조건을 테스트하는지, 어떤 결과가 유추되는지에 대한 정보를 모두 포함하는게 좋아요. 추상적으로요 ㅎㅎ 

```suggestion
    @DisplayName("구간 삭제 함수는 유효한 구간 삭제를 요청할 경우 정상적으로 삭제된다.")
```

예를 들면 이렇게 작성되면 콘솔만 보고도 무슨 테스트가 수행되었는지 알 기 편하겠죠?

Reply:
오우 정말 한눈에 보기 편하겠군요!